article/index.html

```diff
@@ -39,13 +39,13 @@
 const products = [
   { classId: 'likenft154xhw0qyds5pgvsyc7379lnkyvwqkvv2zvmmh2gn5qrewljeqwys2sju6x' },
   { classId: 'likenft1ekvx3en9l0640kv9fd5n5hvltwyfl443ymyu2cnhmalvjfy0cdcqqa3l6h', coupon: 'thewitness' },
-].map(({ classId, coupon }) => {
+].map(({ classId, coupon }, index) => {
   const searchParams = new URLSearchParams(window.location.search);
   const utmSource = searchParams.get('utm_source');
   const url = new URL(`https://liker.land/nft/class/${classId}`);
   url.searchParams.set('utm_medium', 'website');
   url.searchParams.set('utm_campaign', 'thewitness-ad');
-  url.searchParams.set('utm_source', 'thewitness-article');
+  url.searchParams.set('utm_source', `thewitness-article-${index + 1}`);
```

Comment:
Will the index => class id relationship ever change though? Maybe just put class id into a new utm_content?

Reply:
Updated